tests/js/server/shell/shell-any-nondeterministic-noncluster.js

```diff
@@ -213,8 +225,16 @@ function AnySuite () {
       var statsExp = statsExpected(l, n * 200);
       var stats = statsFound(dist);
       assertEqual(stats.count, n * 200);
+      if (Math.abs(stats.average - statsExp.average)
+        >= statsExp.averageStddev * 4) {
```

Comment:
This is a bit inconsistent to me. Above and below we print out the values if the deviation is more than 3 times the expected value, and fail if the deviation is bigger than factor 4.
But here we print if the deviation is factor 4 and also fail at the same threshold
```suggestion
        >= statsExp.averageStddev * 3) {
```
I am not sure what the intention is for this. But if we want the thresholds to be aligned, there are a few places to adjust in this PR. The same if we want the thresholds for printing and failing to be different. :smile: